app/src/app/poll/poll.component.html

```diff
@@ -86,90 +86,99 @@ <h4 *ngIf="displayedResult !== undefined" i18n>Results of round {{ displayedResu
 
   </div>
 
-  <div class="information" *ngIf="!!answer">
-    
-    <ng-template #firstRoundTmpl>
-      <p i18n>
-        This is the first round of the poll.
-        <ng-container *ngIf="!hasCurrentRoundBallot()">
-          You can submit a first ballot, in which case you will become a participant of the poll.
-        </ng-container>
-        In the next rounds, you will be able to change your vote, based on the results of the previous round.
-        The first round will end
-        <ng-container [ngSwitch]="roundDeadlinePassed()">
-          <ng-container *ngSwitchCase="true">
-            as soon as three participants have voted.
-          </ng-container>
-          <ng-container *ngSwitchCase="false">
-            {{ answer.RoundDeadline | nearDate:'inside' }}.
-          </ng-container>
-        </ng-container>
-        <ng-container *ngIf="!hasCurrentRoundBallot()">
-          It will not be possible to join the poll once the first round has ended.
-        </ng-container>
-        Then, each round will last {{ answer.MaxRoundDuration | duration }}, or until all participants have voted,
-        whichever comes first.
-      </p>
-    </ng-template>
+  <div class="simplemenu">
+    <mat-icon class="helpbutton" id="helpbutton">help</mat-icon>
+  </div>
 
-    <ng-container [ngSwitch]="answer.State">
-      <ng-container *ngSwitchCase="'Waiting'">
-        <p i18n>
-          The poll will start {{ answer.Start | nearDate:'inside' }}.
-          You won't be able to vote until that date.
-        </p>
-      </ng-container>
-      <ng-container *ngSwitchCase="'Terminated'">
-        <p i18n>
-          Votes are now closed for this poll.
-          The displayed result is the final one.
-        </p>
-      </ng-container>
-      <ng-container *ngSwitchCase="'Active'">
-        <p *ngIf="answer.CurrentRound > 0 else firstRoundTmpl" i18n>
-          This is the {{ answer.CurrentRound + 1 | ordinal }} round of the poll.
-          You can now change your vote, based on the result of the previous round.
-          If you do not vote before the end of the round,
-          <ng-container [ngSwitch]="answer.CarryForward">
-            <ng-container *ngSwitchCase="false">
-              you will be declared abstentionist.
-            </ng-container>
-            <ng-container *ngSwitchCase="true">
-              your previous vote will be carried forward.
-            </ng-container>
-          </ng-container>
-          The round will end {{ answer.RoundDeadline | nearDate:'inside' }} or when all participants have voted,
-          whichever comes first.
-        </p>
+  <div class="infomenu">
+    <table>
+      <tr>
```

Comment:
No. You're not displaying a table, hence you can not use a table element. Using table to ease your layout is just plain wrong. Instead, you could use modern CSS layout possibilities like [flex](https://www.w3.org/TR/css-flexbox-1/) and [grid](https://www.w3.org/TR/css-grid-1/).